.travis.yml

```diff
@@ -21,7 +21,7 @@ before_install:
   - cd coalahtml/_coalahtml
   - bower install
   - cd -
-  - sudo pip3 install -r requirements.txt
+  - sudo pip3 install -U -r requirements.txt
```

Comment:
Without upgrading, it throws an error saying installed colorama version and required are different. I am not sure why this occurs. I assume only PyPrint requires colorama here. What else could be causing this conflict? 
@Uran198  @AbdealiJK